kernel/constr.ml

```diff
@@ -850,7 +850,7 @@ let map_with_binders g f l c0 = match kind c0 with
 let rec exliftn el c =
   let open Esubst in
   match kind c with
-  | Rel i -> mkRel(reloc_rel i el)
+  | Rel i -> let n = reloc_rel i el in if Int.equal n i then c else mkRel n
```

Comment:
Is ths change actually useful?

Reply:
Useful for the PR no. Useful in general I even don't know since the mkRel are already mostly all shared. It was a change in passing that I did not remove (will remove it).

Similarly, the `esubst.mli` documentation was done in passing and is not directly related to the PR.

Reply:
please drop this code complexification